src/components/config/parse.ts

```diff
@@ -28,8 +28,13 @@ import { parseConfigFromPostman } from './parse-postman'
 import { parseHarFile } from './parse-har'
 import path from 'path'
 import yml from 'js-yaml'
+import { Notification } from '../../interfaces/notification'
 
-export const parseConfig = (configPath: string, type: string): Config => {
+export const parseConfig = (
+  configPath: string,
+  type: string,
+  omitNotifications: boolean
```

Comment:
it's better to make this optional, so we don't need existing callers' signatures.